explorer/ormconfig.json

```diff
@@ -53,10 +53,11 @@
   {
     "name": "ci",
     "type": "postgres",
-    "host": "localhost",
+    "host": "postgres",
```

Comment:
Looks like this change is making current CI fail. Let's make a `ci-gh` key that we use for the new CI, then migrate it back over when we remove the circleci job for `explorer`